pkg/handlers/logout.go

```diff
@@ -2,19 +2,34 @@ package handlers
 
 import (
 	"net/http"
+	"time"
 
+	"github.com/pkg/errors"
 	"github.com/replicatedhq/kots/pkg/logger"
 	"github.com/replicatedhq/kots/pkg/session"
 	"github.com/replicatedhq/kots/pkg/store"
 )
 
 func (h *Handler) Logout(w http.ResponseWriter, r *http.Request) {
-	if r.Header.Get("Authorization") == "" {
+
+	signedTokenCookie, err := r.Cookie("signed-token")
+
+	if err == http.ErrNoCookie {
```

Comment:
what if there is an error but it isn't `http.ErrNoCookie`. should we log it and mark it as an internal server error?

Reply:
As of now the [request](https://go.dev/src/net/http/request.go) Cookie function only returns `nil` or `http.ErrNoCookie` as errors:
```
// Cookie returns the named cookie provided in the request or
// ErrNoCookie if not found.
// If multiple cookies match the given name, only one cookie will
// be returned.
func (r *Request) Cookie(name string) (*Cookie, error) {
	if name == "" {
		return nil, ErrNoCookie
	}
	for _, c := range readCookies(r.Header, name) {
		return c, nil
	}
	return nil, ErrNoCookie
}
```
Would adding a check for other types of errors be to future-proof in case there are others returned in the future?